libobs/obs-output.c

```diff
@@ -145,6 +149,8 @@ obs_output_t *obs_output_create(const char *id, const char *name,
 
 	output->reconnect_retry_sec = 2;
 	output->reconnect_retry_max = 20;
+	output->reconnect_retry_exp =
+		RECONNECT_RETRY_BASE_EXP + (rand_float(0) / 20);
```

Comment:
```suggestion
		RECONNECT_RETRY_BASE_EXP + (rand_float(0) * 0.05);
```
How about writing `* 0.05` so that easily know it's 0.05 second (50 ms) random time.